R/plot_DRTResults.R

```diff
@@ -488,7 +491,7 @@ plot_DRTResults <- function(
                y = summary.pos[2],
                adj = summary.adj,
                labels = label.text[[i]],
-               cex = 0.8 * cex,
+               cex = 0.8,
```

Comment:
I have to admit that this was in a scaler on purpose. However, if it works without `* cex`. Fine with me.

Reply:
There's an annoying difference between how the `cex` option is interpreted inside `text()` and `legend()`, which is different from how it's interpreted in `mtext()` and other plotting functions. In those two functions the docs say:  

> cex: character expansion factor *relative* to current  ‘par("cex")’.

Reply:
OK, perhaps this has also changed or I never understood how it is implemented. Thanks either way.